FileSpecs/Maps.md

```diff
@@ -0,0 +1,206 @@
+| `3`      | u8                 | tileset / background | 2D maps: tileset (`1` or `2`).  3D maps: background image (?) (`0x01`-`0x17`)    |
+| `4`      | u8                 | 3D map marer         | `00` for top-down 2D maps, `01` for first-person 3D maps (dungeons, towns)       |
+| `5`      | u8                 | flags                | See below                                                                        |
+| `6`      | u8                 | song                 | If ≠ `00`, set active song to this one                                           |
+| `7`      | u8                 | width                | in # tiles                                                                       |
+| `8`      | u8                 | height               | in # tiles                                                                       |
+| `9`      | char[30]           | name                 | map name, padded with `0x20`, and `0x00`-terminated                              |
+| `38`     | event[254]         | events               | `10` bytes per [event](Events.md)                                                 |
+| `a14`    | npc[24]            | NPCs                 | NPC list, See below                                                                        |
+| `abc`    | u8[9]              | magic number         | = [`01` `20` `0c` `1e` `18` `3c` `05` `0c` `0c`]                                 |
+| `ac5`... |                    |                      | varies between 2D and 3D maps, see below                                         |
+|          | LabInfo[..]        | labinfo              | 3D only: probably `LAB_DATA` references (?)                                      |
+|          | u8[width * height] | map[0]               | map data                                                                         |
+|          | u8[width * height] | map[1]               | 2D only: overlay map                                                             |
+|          | u8[width * height] | hotspots             | hotspot map.  If hotspot[`pos`] = `n` > 0, then event[`n - 1`] triggers at `pos` |
+|          | u8[..]             | NPC coordinates      | See below                                                                                 |
+
+
+## Map Flags
+
+The map flags likely indicate light sources, whether resting is possible etc.:
+
+| Flag | Name | Exclusivity                                   | Meaning            | Example maps                                                                        |
+|------|------|-----------------------------------------------|--------------------|-------------------------------------------------------------------------------------|
+| 0x01 |      | A: not with `02`,`04`, `20`, not in 3D        |                    | Sir Marillon's Tomb, Family home                                                    |
```

Comment:
The first 3 bits are most likely: indoor, outdoor and dungeon like in Ambermoon. Would also fit with your mentioned maps.

Reply:
That sounds likely; I didn't spot any counter-example to that theory while going through the list.